srcpkgs/esptool/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'esptool'
+pkgname=esptool
+version=3.1
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-pip"
+depends="python3-pyserial python3-ecdsa python3-pyaes
+ python3-bitstring python3-reedsolo"
```

Comment:
Can you put this in same order as https://github.com/espressif/esptool/blob/master/setup.py#L128 ?

And I think it's missing cryptography explicitly?

Given https://github.com/espressif/esptool/blob/master/setup.py#L118, you probably need some additional `checkdepends` too.